.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    target-branch: master
+    reviewers:
+      - stephendolan
+
+  - package-ecosystem: npm
+    directory: "/src/browser-app-skeleton"
+    schedule:
+      interval: daily
```

Comment:
Would it be less hectic if we did like monthly instead?

Reply:
Perhaps... the trouble is that doing it monthly doesn't mean you get one combined PR with the dependency updates, it just means that you get a month's worth of PRs all at once.

If we leave it daily, there's no harm in ignoring them until we've got time, and Dependabot will automatically close and open new ones if dependency versions are changed after a PR is created.

Reply:
After going down the weekly route, it's way less hectic to receive a PR every couple of days than receiving 15 (I'm pretty sure that's the default limit) PR's all out once.